src/js/features/excerpt-generation/classic/index.js

```diff
@@ -87,10 +94,322 @@ const classifaiExcerptData = window.classifaiGenerateExcerpt || {};
+			if ( defaultExcerptContent.length ) {
+				defaultExcerptContent.hide();
+			}
+
+			// Make sure the regenerate button is visible.
+			const existingRegenerateButton = $(
+				'#classifai-excerpt-generation__excerpt-generate-btn'
+			);
+			if ( existingRegenerateButton.length ) {
+				existingRegenerateButton.show();
+			}
+		}
+
+		// Function to show custom field info.
+		function showCustomFieldInfo() {
+			if (
+				! targetFieldSettings ||
+				targetFieldSettings.field_type === 'post_excerpt'
+			) {
+				return;
+			}
+
+			// Get the current value.
+			apiFetch( {
+				path: `/classifai/v1/get-target-field-value/${ postId }`,
+			} )
+				.then( ( result ) => {
+					targetFieldValue = result.value || '';
+
+					// Create custom field info container.
+					const customFieldContainer = $( '<div>', {
+						class: 'classifai-custom-excerpt-info',
+						style: 'margin: 0; padding: 15px; background: #f9f9f9; border: 1px solid #ddd; border-radius: 4px;',
+					} );
+
+					const label = $( '<label>', {
+						text:
+							targetFieldSettings.field_name ||
+							__( 'Custom excerpt', 'classifai' ),
+						style: 'display: block; margin-bottom: 8px; font-weight: 600; color: #23282d;',
+					} );
+
+					const textarea = $( '<textarea>', {
+						id: 'classifai-custom-excerpt',
+						text: targetFieldValue, // Use text() instead of value for textarea.
+						style: 'width: 100%; min-height: 120px; padding: 8px; border: 1px solid #ddd; border-radius: 3px; font-family: inherit; font-size: 14px; line-height: 1.4; resize: vertical; background: #f5f5f5;',
+						readonly: true,
+						placeholder: __(
+							'This field is read-only. The value is stored in a custom field.',
+							'classifai'
+						),
+					} );
+
+					const notice = $( '<div>', {
+						class: 'notice notice-info',
+						style: 'margin-top: 10px;',
+					} ).append(
+						$( '<p>', {
+							html:
+								( targetFieldSettings.field_type === 'acf_field'
+									? __(
+											'This excerpt is stored in an ACF field. To edit it, you can:',
+											'classifai'
+									  )
+									: __(
+											'This excerpt is stored in a custom field. To edit it, you can:',
+											'classifai'
+									  ) ) +
+								'<ul style="margin: 5px 0 0 20px;">' +
+								'<li>' +
+								__(
+									'Use the regenerate excerpt button to regenerate it',
+									'classifai'
+								) +
+								'</li>' +
+								( targetFieldSettings.field_type === 'acf_field'
+									? '<li>' +
+									  __(
+											'Edit the ACF field directly in the post editor or ACF fields panel',
+											'classifai'
+									  ) +
+									  '</li>'
+									: '<li>' +
+									  __(
+											'Edit the custom field directly in the post editor or custom fields panel',
+											'classifai'
+									  ) +
+									  '</li>' ) +
+								'<li>' +
+								__(
+									'Change the target field in',
+									'classifai'
+								) +
+								'<a href="' +
```

Comment:
```suggestion
								' <a href="' +
```

Add space before settings link.

<img width="405" height="68" alt="Image" src="https://github.com/user-attachments/assets/ecd380e0-41e7-4c5b-a2c2-2f3e811ff8a1" />